testsuites/syncgateway/functional/tests/test_channels.py

```diff
@@ -73,6 +76,23 @@ def test_channels_view_after_restart(params_from_base_test_setup, sg_conf_name):
     sg.stop_sync_gateways(cluster_config=cluster_config, url=sg_url)
     sg.start_sync_gateways(cluster_config=cluster_config, url=sg_url, config=sg_conf)
 
+    # Add 1 to Sync Gateway
+    doc_bodies = document.create_docs(doc_id_prefix='doc_new', number=1, channels=seth_user_info.channels)
+    bulk_docs_resp_new = client.add_bulk_docs(url=sg_url, db=sg_db, docs=doc_bodies, auth=seth_session)
+    assert len(bulk_docs_resp_new) == 1
+
+    # Repopulate channel cache with view call
```

Comment:
Maybe more descriptive to say "Changes request to trigger population of channel cache with view call"

Reply:
Done